src/index.html

```diff
@@ -12,7 +12,143 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <div class="stars stars--0">
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+    </div>
+
+    <div class="stars stars--1">
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
```

Comment:
```suggestion
      <a href="#" class="stars__star"></a>
```
And you can use div instead of link